pkgs/development/tools/misc/sccache/default.nix

```
@@ -29,6 +29,7 @@ rustPlatform.buildRustPackage rec {
   buildInputs = [
     openssl
   ] ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```

```suggestion
  ] ++ lib.optional stdenv.isDarwin Security;
```

Please add Security to inputs and iherit it from top level ``inherit (darwin.apple_sdk.frameworks) Security``.